NCATS_Data_Dictionary.md

```diff
@@ -49,6 +49,7 @@ This information is organized by database and collection (table).
 * [emails Collection](#emails-collection)
 * [templates Collection](#templates-collection)
 * [users Collection](#users-collection)
+* [user_reports Collection](#user_reports-collection)
```

Comment:
Keep alphabetical order intact:
```suggestion
* [user_reports Collection](#user_reports-collection)
* [users Collection](#users-collection)
```

Reply:
Taken care of in commit 0277a31978eace2b882e6981e9ec343e34574209